finagle-netty4/src/main/scala/com/twitter/finagle/netty4/Netty4HashedWheelTimer.scala

```diff
@@ -67,7 +67,7 @@ private class HashedWheelTimer(
   self.newTimeout(pendingTasksStat, statsPollInterval.inSeconds, TimeUnit.SECONDS)
 }
 
-private object HashedWheelTimer {
+object HashedWheelTimer {
```

Comment:
thinking about it more, I think that we should probably make `instance` private, and just have a stop method on this object.  that way people can't pass around HashedWheelTimer willy-nilly and accidentally get confused and stop it.  you have to know that you're stopping the default one.

Reply:
Yes, that was one of my ideas. Done